back/apps/core-fca-low/src/views/interaction.ejs

```diff
@@ -34,28 +34,42 @@
           </div>
         <% } %>
         <form action="/api/v2/redirect-to-idp" method="POST" class="interaction-main-content-form" name="email-check">
-          <label class="fr-label fr-mt-1v" for="email-input">
-            Email professionnel
-            <span class="fr-hint-text">Format attendu : nom@domaine.fr</span>
-          </label>
+          <div class="<% if (locals.error) { %> fr-input-group fr-input-group--error<% } %>">
```

Comment:
suggestion: There is currently only one known error type, invalid_email. We should make this explicit here before introducing a generic implementation.